### DISCUSSION TITLE
Make Tweens easier to reuse

### DISCUSSION THREAD

**Discussion by timothyqiu** [godotengine/godot-proposals#5380] [General Discussions]
A `Tween` (or `SceneTreeTween` in 3.5) is invalidated when finished. There seems no way to restart such a tween.

The current workaround seems to be connecting the `finished` signal to its `stop` method. This feels hacky, a dedicated property might be better.

**Comment by SilencedPerson** [comment]
There is `set_loops()`?
Doesn't that suffice?

**Reply by SilencedPerson** [reply]
You could also use tween_callback() if you wrap your tween in function.

**Reply by timothyqiu** [reply]
I mean, setup a complex tween animation, then play it when triggered. e.g. whenever the space bar is pressed.

Currently, I have to setup the tween everytime the space bar is pressed. Or connect the `finished` signal to `stop` like said in OP.
